File: |-- src_R/| |-- 03_akar_kuadrat.R
cat("Program Menghitung Akar Persamaan Kuadrat\n")

a <- 1
b <- -5
c <- 6

diskriminan <- b^2 - 4 * a * c

if (diskriminan < 0) {
  cat("Persamaan hanya memiliki akar-akar imajiner\n")
} else {
  akar1 <- (-b + sqrt(diskriminan)) / (2 * a)
  akar2 <- (-b - sqrt(diskriminan)) / (2 * a)

  cat("Akar pertama =", format(akar1, nsmall = 3), "\n")
  cat("Akar kedua =", format(akar2, nsmall = 3), "\n")
}

File: |-- src_python/| |-- 03_akar_kuadrat.py
print("Program Menghitung Akar Persamaan Kuadrat")

a = float(input("Masukkan nilai a: "))
b = float(input("Masukkan nilai b: "))
c = float(input("Masukkan nilai c: "))

diskriminan = b**2 - 4 * a * c

if diskriminan < 0:
    print("Persamaan hanya memiliki akar-akar imajiner")
else:
    akar1 = (-b + diskriminan**0.5) / (2 * a)
    akar2 = (-b - diskriminan**0.5) / (2 * a)

    print("Akar pertama =", format(akar1, ".3f"))
    print("Akar kedua =", format(akar2, ".3f"))
